theme.go

```diff
@@ -13,12 +14,130 @@ import (
 	"github.com/charmbracelet/glamour/ansi"
 )
 
+// Themes return the list of themes.
+func Themes() map[string]Theme {
```

Comment:
How come this is a function rather than a `var`?

Reply:
just so its modifiable... but on second thought that might not be worth it